redisai/model.go

```diff
@@ -17,58 +18,60 @@ type ModelInterface interface {
 	SetDevice(device string)
 	Backend() string
 	SetBackend(backend string)
+	Tag() string
+	SetTag(tag string)
 }
 
 func modelSetFlatArgs(keyName, backend, device string, inputs, outputs []string, blob []byte) redis.Args {
-	args := redis.Args{}.Add(keyName, backend, device)
-	if len(inputs) > 0 {
-		args = args.Add("INPUTS").AddFlat(inputs)
-	}
-	if len(outputs) > 0 {
-		args = args.Add("OUTPUTS").AddFlat(outputs)
-	}
-	args = args.Add("BLOB")
-	args = args.Add(blob)
-	return args
+	modelInterface := implementations.NewModel(backend, device)
```

Comment:
avoid creating a structure just to retrieve redis.Args ( please see the https://github.com/dengliming/redisai-go/pull/1 )

Reply:
Thanks for helping. Merged.